core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/function/FilterHandler.java

```diff
@@ -50,6 +51,13 @@
             operation.setEdges(edgeMap);
         }
 
+        final FunctionValidator<Filter> validator = new FunctionValidator<>();
```

Comment:
Rather than creating a new instance of this for every query, the validator could be a 'private final' field.